website/docs/plugin/framework/migrating/resources/import.mdx

```diff
@@ -0,0 +1,197 @@
+---
+page_title: 'Resources - Import: Migrating from SDKv2 to the Framework'
+description: >-
+  Practitioners use the import command to let Terraform manage existing infrastructure resources.
+  Migrate import functions from SDKv2 to the plugin Framework.
+---
+
+# Import
+
+Practitioners can use the [`terraform import` command](https://www.terraform.io/cli/commands/import) to let Terraform
+begin managing existing infrastructure by importing an existing resource into their Terraform project's state. A
+resource's importer function implements the logic to add a resource to Terraform's state. Refer to
+[Resources - Import](/plugin/framework/resources/import) in the Framework documentation for details.
+
+This page explains how to migrate import functions from SDKv2 to the plugin Framework.
+
+## SDKv2
+
+In SDKv2, the `Importer` field on the `schema.Resource` defines how the provider imports resources to Terraform's
+state. The following example implements resource import with SDKv2.
+
+```go
+func resourceExample() *schema.Resource {
+    return &schema.Resource{
+        Importer: &schema.ResourceImporter{
+            StateContext: StateContextFunc,
+        },
+        /* ... */
+```
+
+The `StateContextFunc` is the function called to import a resource into Terraform state. Any operations that
+are needed to import the resource take place within this function and may result in the mutation of the `ResourceData`
+that is passed into the function. The return value is a slice of `schema.ResourceData`. This slice, which may be simple, likemight be as simple as returning the `ResourceData` that was passed into the function, or it may involve multipleinvolve fan out to multiple resources, such as AWS security groups.
+
+## Framework
+
+In the Framework, you implement the `ResourceWithImportState` interface on your `resource.Resource` type to allow
+users to import a given resource.
+
+The following code shows how you define an `ImportState` function with the Framework.
+
+```go
+func (r *resourceExample) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
+    /* ... */
+}
+```
+
+The `ImportState` function includes a `resource.ImportStateResponse`, which you use to mutate and set your resource's
+state by mutating the state.
```

Comment:
The wording here is a bit awkward, but I'm not sure how to fix it. ... it seems to be saying that you mutate your resource's state by mutating the state ...?

Reply:
I've reworded as follows:

> The `ImportState` function includes a `resource.ImportStateResponse`, which you use to set your resource's state.